rtsp.go

```diff
@@ -438,16 +450,16 @@ func (rc *rtspCamera) initH265(session *description.Session) (err error) {
 		}
 
 		for _, nalu := range au {
-			lastImage, err := rc.rawDecoder.decode(nalu)
+			output, err := rc.rawDecoder.decode(nalu)
```

Comment:
Dumb question -- definitely paged out a lot of stuff over the last couple weeks. In what circumstance does `err != nil` but `output == nil`? Is this related to that linting thing I said you ought not worry about?